### PR TITLE
#895 Add Swin CPU implementation with HF demo, torch demo

### PR DESCRIPTION
**Pull request by saichandax**
This branch contains the following:

CPU implementation of the Swin model from huggingface transformers. It includes modeling of Swin along with its dependent libraries.
This branch has the demo for torch_cpu and hugging face reference model, along with the test_torch_model file to compare the two.

### CODE REVIEW

**Review by tt-rkim — CHANGES_REQUESTED** [review]
I'm confused on this whole PR. What's the point? There is no TT code.

Are these demos? Or just merging in Pytorch references first?

Also, the files under `demo/` are unnecessary to me.

**Review by ashayestehmanesh — CHANGES_REQUESTED** [review]

**Review by ashayestehmanesh — DISMISSED** [review]
@saichandax 

Please make sure of 3 things before merging
1. all the tests work
2. squash your commits into one commit
3. all the code you are committed is used, do not merge any unused code. 

Thank you :)

**Review by farbabi — APPROVED** [review]

**Review by tt-rkim — APPROVED** [review]

**Pull request merged**
